src/services/fraud-prover.service.ts

```diff
@@ -306,7 +306,14 @@ export class FraudProverService extends BaseService<FraudProverOptions> {
             if (
               err
                 .toString()
-                .includes('Function must be called during the correct phase.')
+                .includes(
+                  'Function must be called during the correct phase.'
+                ) ||
+              err
+                .toString()
+                .includes(
+                  '46756e6374696f6e206d7573742062652063616c6c656420647572696e672074686520636f72726563742070686173652e'
```

Comment:
This is the byte representation of `Function must be called during the correct phase`. Some providers only return it in bytes :-(

Reply:
Maybe there should be a helper fn that checks for both possibilities and then can be used in each place that the revert message is used, can use `Buffer.from(message, 'ascii').toString('hex')`